src/components/views/settings/tabs/user/VoiceUserSettingsTab.js

```diff
@@ -215,19 +218,34 @@ export default class VoiceUserSettingsTab extends React.Component {
                     {speakerDropdown}
                     {microphoneDropdown}
                     {webcamDropdown}
-                    {SettingsStore.getValue(UIFeature.ShowAdvancedCallSettings) && <>
-                        <SettingsFlag name='VideoView.flipVideoHorizontally' level={SettingLevel.ACCOUNT} />
-                        <SettingsFlag
-                            name='webRtcAllowPeerToPeer'
-                            level={SettingLevel.DEVICE}
-                            onChange={this._changeWebRtcMethod}
-                        />
-                        <SettingsFlag
-                            name='fallbackICEServerAllowed'
-                            level={SettingLevel.DEVICE}
-                            onChange={this._changeFallbackICEServerAllowed}
-                        />
-                    </>}
+                    {SettingsStore.getValue(
```

Comment:
Please minimise any changes to existing code.  Is you IDE set to wrap lines with a short line length?  Our standard for line length wrapping is 160 characters.
Is there a difference between SettingsStore.getValue(UIFeature.ShowAdvancedCallSettings) && <>
and SettingsStore.getValue(UIFeature.ShowAdvancedCallSettings) ? ... : null ?  As the bool && <> is used throughout the existing code.

Reply:
&& did not work at the time of making many tweaks but now it works so have changed null to && syntax. I wanted to use this , at that time it wasnt working, now it does work which is good.